lldb/packages/Python/lldbsuite/test/lldbplatformutil.py

```diff
@@ -97,6 +101,9 @@ def finalize_build_dictionary(dictionary):
             dictionary = {}
         dictionary["OS"] = "Android"
         dictionary["PIE"] = 1
+    elif target_is_remote_linux():
```

Comment:
For consistency, I think we should do this for all linux systems, not just remote ones. (Ideally, I would do it for all systems overall, but I can understand if you're reluctant to do that.)

You can use `getPlatform` to get the platform/os name for both local and remote runs.